<define-tag description>Pufferüberlauf</define-tag>
<define-tag moreinfo>
<p>Sebastian Schmidt entdeckte einen Pufferüberlauf-Fehler in Pythons
getaddrinfo-Funktion, der es einer von einem entfernten Angreifer über DNS
gelieferte IPv6-Adresse erlauben könnte, Speicher auf dem Stack zu
überschreiben.</p>

<p>Dieser Fehler ist nur in Python 2.2 und 2.2.1 vorhanden, und nur, wenn
IPv6-Unterstützung aktiviert ist. Das python2.2-Paket in Debian woody erfüllt
diese Bedingungen (das »python«-Paket tut dies nicht).</p>

<p>Für die stable Distribution (woody) wurde dieser Fehler in
Version 2.2.1-4.3 behoben.</p>

<p>Die unstable Distribution (sid) ist nicht von diesem Fehler betroffen.</p>

<p>Wir empfehlen Ihnen, Ihr python2.2-Paket zu aktualisieren.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2004/dsa-458.data"
#use wml::debian::translation-check translation="1.2"
# $Id$
